gr-utils/python/modtool/templates/gr-newmod/docs/doxygen/other/group_defs.dox

```diff
@@ -1,5 +1,5 @@
 /*!
- * \defgroup block GNU Radio HOWTO C++ Signal Processing Blocks
+ * \defgroup block GNU Radio C++ Signal Processing Blocks
```

Comment:
The HOWTO gets transfomed into OOT name by modtool. The correct fix is to ignore gr-newmod/ entirely; it should not be in the manual.

Reply:
For that you need to add 
```
@abs_top_srcdir@/gr-utils/python/modtool/templates/gr-newmod/docs/doxygen \ 
@abs_top_builddir@/gr-utils/python/modtool/templates/gr-newmod/docs/doxygen \
```
here, right:
https://github.com/gnuradio/gnuradio/blob/2003dda0593cc000af186532e90e2b77aa4d383e/docs/doxygen/Doxyfile.in#L729-L730

Reply:
That looks right.